source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRContentStackView.mm

```diff
@@ -52,6 +56,9 @@ - (instancetype)initWithFrame:(CGRect)frame attributes:(nullable NSDictionary<NS
 - (instancetype)initWithFrame:(CGRect)frame
 {
     self = [self initWithFrame:CGRectMake(0, 0, frame.size.width, frame.size.height) attributes:nil];
+    if (self) {
+        _hiddenSubviews = [[NSMutableSet alloc] init];
```

Comment:
>_hiddenSubviews = [[NSMutableSet alloc] init]; [](start = 8, length = 46)

It's odd to put this initialization here but not in any of the other initializers. If this view is constructed via initWithCoder: or initWithFrame:attributes:, then `_hiddenSubviews` will never be constructed.

The usual pattern in Objective-C is to identify one initializer as the "designated initializer", and make sure that all initialization paths flow through it.

[http://www.timekl.com/blog/2014/12/09/objective-cs-designated-secret/](http://www.timekl.com/blog/2014/12/09/objective-cs-designated-secret/)